### DISCUSSION TITLE
Filter by "HasManyThrough" relationships

### DISCUSSION THREAD

**Discussion by pmartelletti** [filamentphp/filament#8469] [General]
This PR added the ability to search via "HasManyThrough" relationships using the Select filter in v2. However, in v3 this has been removed and `relationship` method only accepts one of:

```
BelongsTo | BelongsToMany | \Znck\Eloquent\Relations\BelongsToThrough
```

The fix is a simple one (at least, from what I've attempted on my local machine, and based on that PR). However, before raising the PR, wanted to double-check if there was any particular reason why this was removed from v3? I've looked through the discussion and issues and couldn't find any valid reason so far.

Thanks.

**Comment by joshembling** [comment]
Hi @pmartelletti @danharrin, is there any update on this?
